### ISSUE TITLE
Bridge creation (and as such, deployment) fails if empty in Debian Buster KVM host (bridge-utils 1.6)

### ISSUE THREAD

**Issue by tmartincpp** [OpenNebula/one#3522]
**Description**
In a Debian Buster KVM host, if there is no bridge already configured on host, the auto-configuration of the bridges will fail, and as such deployment will fail too. The bug doesn't occur if there is already at least one bridge active on the KVM host.

**To Reproduce**
Deploy a VM with one or multiple bridged NIC on a KVM Debian Buster host which doesn't have at least one bridge currently active.

**Expected behavior**
The bridges are created and the VM is deployed.

**Details**
 - Affected Component: Scheduler
 - Hypervisor: KVM
 - Version: 5.8.1

**Additional context**
It seems to be because the output of brctl changed on Debian Buster when there is no active bridge (bridge-utils 1.6 VS bridge-utils 1.5)

Debian 9 Stretch (and lower):

> root@debian9:\~# brctl show
> bridge name     bridge id               STP enabled     interfaces
> root@debian9:\~# 

Debian 10 Buster:

> root@debian10:\~# brctl show
> root@debian10:\~# 

Note that if there is a bridge on the host, the description row will be printed along the brige row, like it was before.

Error log:
```
Mon Jul 15 15:13:12 2019 [Z0][VMM][I]: Command execution fail: /var/tmp/one/vnm/bridge/pre [...REDACTED...]
Mon Jul 15 15:13:12 2019 [Z0][VMM][I]: /var/tmp/one/vnm/vlan.rb:166:in `get_bridges': undefined method `each' for nil:NilClass (NoMethodError)
Mon Jul 15 15:13:12 2019 [Z0][VMM][I]: from /var/tmp/one/vnm/vlan.rb:36:in `activate'
Mon Jul 15 15:13:12 2019 [Z0][VMM][I]: from /var/tmp/one/vnm/bridge/pre:29:in `<main>'
Mon Jul 15 15:13:12 2019 [Z0][VMM][I]: ExitCode: 1
Mon Jul 15 15:13:12 2019 [Z0][VMM][I]: Failed to execute network driver operation: pre.
Mon Jul 15 15:13:12 2019 [Z0][VMM][E]: Error deploying virtual machine: bridge: -
```

## Progress Status
- [ ] Branch created 
- [ ] Code committed to development branch
- [ ] Testing - QA
- [ ] Documentation 
- [ ] Release notes - resolved issues, compatibility, known issues
- [ ] Code committed to upstream release/hotfix branches
- [ ] Documentation committed to upstream release/hotfix branches

**Comment by vholer** [comment]
Also related to brctl #3613

**Comment by vholer** [comment]
Fixed by #3613
